molecule/default/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+collections:
+  - ansible.posix
+
+roles:
+  - name: geerlingguy.postgresql
```

Comment:
In the readme I recommend [ANXS.postgresql](https://github.com/ANXS/postgresql).Can we use it here too? There is no special reason for it, just for consistency.